gpu4pyscf/scf/tests/test_rhf.py

```diff
@@ -264,11 +265,23 @@ def test_rhf_d4(self):
         e_ref = -151.09634038447925
         assert np.abs(e_tot - e_ref) < 1e-5
 
+    def test_chkfile(self):
+        ftmp = tempfile.NamedTemporaryFile(dir = pyscf.lib.param.TMPDIR)
+        mf = scf.RHF(mol)
+        mf.chkfile = ftmp.name
+        mf.kernel()
+        dm_stored = mf.make_rdm1(mf.mo_coeff, mf.mo_occ)
+        dm_stored = cupy.asnumpy(dm_stored)
+
+        mf_copy = scf.RHF(mol)
+        mf_copy.chkfile = ftmp.name
+        dm_loaded = mf_copy.init_guess_by_chkfile()
+        assert np.allclose(dm_stored, dm_loaded, atol = 1e-14)
```

Comment:
1e-14 is probably too strict. 1e-8 or 1e-9 should be enough for this test.

Reply:
I added a comment about it. The two density matrices are not from two SCF calculations, but from the MO coefficients before storage and after storage and reload. This threshold suppose to be zero, I just allow some numerical noise.